google/cloud/datacatalog/v1beta1/datacatalog_gapic.yaml

```diff
@@ -127,6 +127,33 @@ interfaces:
     retry_codes_name: non_idempotent
     retry_params_name: default
     timeout_millis: 60000
+    samples:
+      standalone:
+      - value_sets: [data_catalog_search]
+        region_tag: data_catalog_search
```

Comment:
Nit: Let's start our region tags with `datacatalog_` so that our sample tracking infrastructure can link it to the right product.

Reply:
Done